### ISSUE TITLE
Permission denied (publickey)

### ISSUE THREAD

**Issue by mittalyashu** [SublimeGit/SublimeGit#249]
Showing this message.
![image](https://user-images.githubusercontent.com/29014463/31732653-dd577ce6-b456-11e7-9968-4d444a649d9e.png)

**Comment by mittalyashu** [comment]
I have figured out the problem. As you have mentioned in your [documentation](https://sublimegit.readthedocs.io/en/latest/tutorial.html#sharing-our-project-with-the-world).

![image](https://user-images.githubusercontent.com/29014463/31734721-44fbc6d0-b45d-11e7-8c2c-1e2e69ace9bc.png)

In the above image, it is showing that we need to paste the link in the `SSH` version.
For example: `git@github.com:mittalyashu/learning.git`

I have tried using it many times but it didn't work. Then I tried using the `https` version, it works.
For example `https://github.com/mittalyashu/sublimeGit.git`
